src/main/java/com/avengers/gamera/dto/article/ArticleGetDto.java

```diff
@@ -0,0 +1,22 @@
+package com.avengers.gamera.dto.article;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.OffsetDateTime;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class ArticleGetDto {
```

Comment:
For Dtos, do we really need setters? Generally speaking, we should always try to make objects immutable. If the Dto/Object should be immutable, just take a look at @Value annotation of lombok.

Reply:
mapper needs setter to convert from Article to ArticleGetDto

Reply:
You can use builder instead of setter, think more about encapsulation

Reply:
fixed